Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -0,0 +1,202 @@
+// 
+
+import UIKit
+
+final class CalculatorViewController: UIViewController {
+    
+    @IBOutlet weak var operatorLabel: UILabel!
+    @IBOutlet weak var operandLabel: UILabel!
+    @IBOutlet weak var calculateStackView: UIStackView!
+    @IBOutlet weak var calculateScrollView: UIScrollView!
+    private var isCalculated: Bool = false
+    private var expression: [String] = []
+    private var calculateOperand: String = Symbol.zero {
+        didSet {
+            operandLabel.text = calculateOperand
+        }
+    }
+    private var calculateOperator: String = Symbol.blank {
+        didSet {
+            operatorLabel.text = calculateOperator
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    private func setupOperandAndOperator() {
+        resetOperand()
+        resetOperator()
+    }
+    
+    @IBAction private func didTapACButton() {
+        calculateStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+        resetOperand()
+        resetOperator()
+    }
+    
+    @IBAction private func didTapCEButton() {
+        if isCalculated == false {
+            calculateOperand = Symbol.zero
+        }
+    }
+    
+    @IBAction private func didTapChangeSignButton() {
+        guard calculateOperand != Symbol.zero else { return }
+        
+        guard let calculateNumberFirst = calculateOperand.first else { return }
+        
+        if calculateNumberFirst == Character(Symbol.minus) {
+            calculateOperand.removeFirst()
+        } else {
+            calculateOperand = Symbol.minus + calculateOperand
+        }
+    }
+    
+    @IBAction private func didTapResultButton() {
+        guard !isCalculated, calculateOperator != Symbol.blank else { return }
+        guard let calculatedNumber = operandLabel.text?.removeComma() else { return }
+        guard let number = Double(calculatedNumber) else { return }
+        
+        addExpressionAndCalculateItem(sign: calculateOperator,
+                                      number: "\(number)",
+                                      operand: calculateOperand)
+        
+        var formula = ExpressionParser.parse(from: expression.joined(separator: Symbol.blank))
+        
+        let result = formula.result()
+        
+        if result.isNaN {
+            calculateOperand = Symbol.nan
+        } else {
+            calculateOperand = NumberFormatter.convertToString(fromDouble: result)
+        }
+        
+        isCalculated = true
+        resetOperator()
+        expression.removeAll()
+    }
+    
+    @IBAction private func didTapOperatorButton(_ sender: UIButton) {
+        guard let operatorSign = sender.currentTitle else { return }
+        guard calculateOperand != Symbol.zero || calculateOperator != Symbol.blank else { return }
+        guard calculateOperand != Symbol.zero || calculateOperator == Symbol.blank else {
+            calculateOperator = operatorSign
+            return
+        }
```

Comment:
공통된 조건이 있으니 아래와 같이 처리할 수 있겠네요 🙂
```suggestion
        guard calculateOperand != Symbol.zero else {
           if  calculateOperator != Symbol.blank {
               calculateOperator = operatorSign
           }
           return
        }
```